### PR TITLE
update out of hours support instructions

### PR DESCRIPTION
**Pull request by Slimleashma**

### CODE REVIEW

File: Safety/out-of-hours-support.rst [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+.. _`#auxtel-operations`: https://lsstc.slack.com/archives/CLQ63T89L
+.. _`#auxtel-virtual-control-room`: https://lsstc.slack.com/archives/C01K4M6R4AH
+.. _`#rubin-observing`: https://lsstc.slack.com/archives/C04HULH5HHD
+
+####################
+Out of hours support
+####################
+
+The Out of Hours support call list is managed in a 1Password vault named "Emergency Call List".
+All observing specialists are automatically enrolled in the vault once they have a 1Password account.
```
Comment (patrickingraham) [inline Safety/out-of-hours-support.rst:32]: this isn't quite true...  Anastasia or myself need to add them manually. So maybe modify the text so as they get an 1password account from IT, then request access from myself or Anastasia.

File: Safety/out-of-hours-support.rst [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+^^^^^^^^^
+
+- If a fault happens during the night, or some new behavior appears in the system, observers and summit staff should attempt to troubleshoot to the extent of their knowledge.
+
+- Ask for help on Slack:
+    - `#auxtel-operations`_
+    - `#auxtel-virtual-control-room`_
+    - `#rubin-observing`_
+
+- If no response is received from Slack, and summit staff have run out of ideas or need assistance, activate the Emergency Call List.
```
Comment (patrickingraham) [inline Safety/out-of-hours-support.rst:53]: activate  == consult ??

I think you should remind people to first consult the confluence page about people's availability before calling. The link to the confluence page is also found inside the vault.

File: Safety/out-of-hours-support.rst [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+    Leave the equipment in the safest configuration possible, if you have the knowledge to do so. 
+
+.. _out-of-hours-support-calling-instructions:
+
+Calling instructions
+^^^^^^^^^^^^^^^^^^^^
+
+- Refer to the :ref:`instructions <summit-contact-numbers-using-the-control-room-phones>` on how to use the summit control rooms and dial an outside line. 
+
+- The Rubin Control room number appears on a caller ID as +1 520-318-8800.
```
Comment (patrickingraham) [inline Safety/out-of-hours-support.rst:70]: This is if the person's cell is based in the US. Presumably the number is different if you have a Chilean phone?

**Review by patrickingraham — CHANGES_REQUESTED** [review]
made a few comments.

File: Safety/out-of-hours-support.rst (hunk 1 above)
Comment (Slimleashma) [reply Safety/out-of-hours-support.rst:32]: 👍

File: Safety/out-of-hours-support.rst (hunk 3 above)
Comment (Slimleashma) [reply Safety/out-of-hours-support.rst:70]: Need to test. I can call myself when I am at summit.

File: Safety/out-of-hours-support.rst (hunk 2 above)
Comment (patrickingraham) [reply Safety/out-of-hours-support.rst:53]: That's correct... but you can say that there is a confluence link in the 1password vault.
The confluence page will be updated more than what is inside the vault (sadly).

Comment (Slimleashma) [reply Safety/out-of-hours-support.rst:53]: 👍

File: Safety/out-of-hours-support.rst (hunk 3 above)
Comment (Slimleashma) [reply Safety/out-of-hours-support.rst:70]: 👍

**Review by patrickingraham — APPROVED** [review]
Looks great! Thanks for your efforts!

**Pull request merged**